### PR TITLE
May 2022 Updates to Challenge 00

### PR DESCRIPTION
**Pull request by mikaelweave**
Changed the README in the repository root and in Challenge-00.

**Base README** updates:
- Retooled language to hint to the workspace / service model
- Removed links about prerequisite knowledge since it's in Challenge-00
- Updated language and formatting to better conform to Microsoft's writing standards.

**Challenge 00** updates:
- Changed all references (explicit or implicit) to open-source tools and moved to AHDS focus in these areas
- Added resource section at the bottom to help learners who need to brush up on fundamentals
- Updated language and formatting to better conform to Microsoft's writing standards.

### CODE REVIEW

File: Challenge-00 - Check prerequisites and configure environment/Readme.md [hunk 1]
```diff
@@ -1,47 +1,84 @@
-# Challenge-00 - Welcome!
+# Challenge-00 - Check Prerequisites and Configure Environment
 
-## Introduction 
-In this preliminary challenge, you will be checking your Azure Subscription and other pieces of your environment to make sure that all prerequisites are met before starting Challenge-01.
+## Introduction
 
-## Background 
-The Azure Health Data Services Workshop presents a series of challenges to help learners build skills in deploying and configuring [FHIR service](https://docs.microsoft.com/en-us/azure/healthcare-apis/fhir/overview), [FHIR-Proxy](https://github.com/microsoft/fhir-proxy), and supporting tools. After completing the workshop challenges, learners will be ready to start building the system architectures featured on __[Microsoft Health Architectures](https://microsoft.github.io/health-architectures/)__.
+In this preliminary challenge, you will be checking your basic knowledge, Azure environment, and other prerequisites of the workshop to ensure you are ready for Challenge-01 and beyond.
```
Comment (v-pbartley) [inline Challenge-00 - Check prerequisites and configure environment/Readme.md:5]: I would remove "checking your basic knowledge".

File: Challenge-00 - Check prerequisites and configure environment/Readme.md [hunk 2]
```diff
@@ -1,47 +1,84 @@
...
 
-+ The tasks require learners to locate information in technical documentation and **resolve issues independently**.
+## What do we want learners to gain from the Azure Health Data Services Workshop?
 
-+ Working on the challenges will give learners a strong concept of how the components fit together. With this knowledge, learners will be able to **deploy Microsoft Health & Life Sciences (HLS) tools in real-world production environments**.  
+In general, we want learners to walk away with a sense of confidence in deploying, configuring, and implementing Microsoft's health data solutions.
 
-## Prerequisite Knowledge for the Azure Health Data Services Workshop
++ In completing this workshop, learners will know the role of Microsoft's health data services.
```
Comment (v-pbartley) [inline Challenge-00 - Check prerequisites and configure environment/Readme.md:21]: suggestion - change "know the role of" to "know the capabilities of"

File: Challenge-00 - Check prerequisites and configure environment/Readme.md [hunk 3]
```diff
@@ -1,47 +1,84 @@
...
++ Learners will know how to **deploy**, **ingest**, **transform**, and **connect health data** using the Azure health data platform.
++ The tasks require learners to locate information in technical documentation and **resolve issues independently**. This workshop is meant to have some challenge to ensure you learn about Microsoft's healthcare platform.
```
Comment (v-pbartley) [inline Challenge-00 - Check prerequisites and configure environment/Readme.md:23]: Suggestion - change "This workshop is meant to have some challenge..." (per Trent, we don't want to emphasize the difficulty) to "This workshop is meant to give learners practice retrieving information from Microsoft documentation resources."

File: Challenge-00 - Check prerequisites and configure environment/Readme.md [hunk 4]
```diff
@@ -1,47 +1,84 @@
...
++ Working on the challenges will give learners a strong concept of how the components fit together. With this knowledge, learners will setup to **use Microsoft Health & Life Sciences (HLS) tools in real-world solutions**.
```
Comment (v-pbartley) [inline Challenge-00 - Check prerequisites and configure environment/Readme.md:24]: change "learners will setup to..." to "learners will be set up to..."

File: Challenge-00 - Check prerequisites and configure environment/Readme.md [hunk 5]
```diff
@@ -1,47 +1,84 @@
...
 
-+ A solid foundation in **Azure fundamentals**. If you are new to Microsoft Azure, please visit [here](https://docs.microsoft.com/en-us/learn/paths/az-900-describe-cloud-concepts/) to learn more about the platform.
+## Prerequisite Knowledge for the Azure Health Data Services Workshop
 
-+ Familiarity with the **FHIR R4 data standard** and general health data interoperability. Please visit [here](https://hl7.org/fhir/R4/) for more information about the FHIR R4 specification.
+These concepts will be necessary to understanding and completing the challenges in this workshop. If you are unfamiliar with any of these areas, review the [More Resources section](#more-resources). Time spent upfront ensuring you have some base knowledge about the technologies used in this workshop will ensure you will be able to gain *actionable experience* for real-world scenarios you will face.
```
Comment (v-pbartley) [inline Challenge-00 - Check prerequisites and configure environment/Readme.md:28]: change - "necessary to understanding and completing..." to "necessary for completing..."

change "base" - "basic"

File: Challenge-00 - Check prerequisites and configure environment/Readme.md [hunk 6]
```diff
@@ -1,47 +1,84 @@
...
 
-+ Experience with making **REST API calls** using [Postman](https://www.postman.com/api-platform/api-testing/) or a similar API testing utility.
++ A solid foundation in **Azure fundamentals** and basic knowledge of **Azure Active Directory**.
++ Familiarity with **FHIR®**  and the solutions it solves versus legacy health data formats.
```
Comment (v-pbartley) [inline Challenge-00 - Check prerequisites and configure environment/Readme.md:31]: change - "the solutions it solves" to "the solutions it provides"

**Comment by mikaelweave** [comment]
Thanks @v-pbartley! Changes made per your feedback.

**Pull request merged**